__TESTS__/unit/url/url.test.ts

```diff
@@ -94,4 +94,16 @@ describe('Tests for URL configuration', () => {
 
     expect(url).toEqual(`https://res.cloudinary.com/demo/image/upload/s--${signature}--/c_crop,w_100/sample`);
   });
+
+  it('Should include query params', function () {
+    const image = createNewImage('sample', {cloudName: 'demo'}, {queryParams: {"_i": "abcde", "_z": 1234, "_t": false}});
+    const url = image.toURL();
+    expect(url).toEqual(`https://res.cloudinary.com/demo/image/upload/sample?_i=abcde&_z=1234&_t=false`);
+  });
+
+  it('Should include query params with analytics', function () {
+    const image = createNewImage('sample', {cloudName: 'demo'}, {analytics: true, queryParams: {"_i": "abcde"}});
+    const url = image.toURL();
+    expect(url).toEqual(`https://res.cloudinary.com/demo/image/upload/sample?_i=abcde&_a=E`);
```

Comment:
`_a=E` is not the best test (`E` stands for error) it would be better to have a valid/mocked analytics token.

Reply:
thanks, fixed